modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -118,8 +118,102 @@ definition {
 		}
 	}
 
-	@description = "LPS-178192. Verifies Expanded Text effects toggle can be enabled."
-	@priority = 4
+	@description = "LPS-178192."
+	@priority = 5
+	test CanBeDisabledByInstanceSettings {
+		property portal.acceptance = "true";
+
+		task ("Given user with admin privileges") {
+			Navigator.openURL();
+		}
+
+		task ("And “Enable Accessibility Menu” is enabled") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "Instance Settings");
+
+			Click(
+				key_itemName = "Accessibility",
+				locator1 = "ListGroupItem#ITEM_TEXT");
+
+			AssertChecked.assertCheckedNotVisible(
+				checkboxName = "Enable Accessibility Menu",
+				locator1 = "Checkbox#ANY_CHECKBOX");
+		}
+
+		task ("When turn off Enable Accessibility Menu") {
+			Uncheck.uncheckToggleSwitch(
+				checkboxName = "Enable Accessibility Menu",
+				locator1 = "Checkbox#ANY_CHECKBOX");
+
+			Click(locator1 = "Button#UPDATE");
+
+			Alert.viewSuccessMessage();
+
+			Refresh();
+		}
+
+		task ("Then Accessibility Menu cannot be accessed") {
+			AccessibilityMenu.goToAccessibilityMenuQuickLink();
+
+			AssertElementNotPresent(
+				key_link = "Open Accessibility Menu",
+				locator1 = "AccessibilityMenu#QUICK_LINK");
+		}
+	}
+
+	@description = "LPS-178192. Verifies it can be enabled by Site Settings."
+	@priority = 3
+	test CanBeEnabledBySiteSettings {
+		property portal.acceptance = "false";
+
+		task ("Given Accessibility Menu is disabled") {
+			Navigator.openURL();
+
+			AccessibilityMenu.deactivateAccessibilityMenu(portlet = "Instance Settings");
+		}
+
+		task ("When navigate to Site Settings > Accessibility And turn on Enable Accessibility Menu") {
+			Navigator.openURL();
+
+			ProductMenu.gotoPortlet(
+				category = "Configuration",
+				portlet = "Site Settings");
+
+			Click(
+				key_itemName = "Accessibility",
+				locator1 = "ListGroupItem#ITEM_TEXT");
+
+			if (!(IsElementPresent(checkboxName = "Enable Accessibility Menu", locator1 = "Checkbox#ANY_CHECKBOX_CHECKED"))) {
+				Check.checkNotVisible(
+					checkboxName = "Enable Accessibility Menu",
+					locator1 = "Checkbox#ANY_CHECKBOX");
```

Comment:
couldn't this be replaced with your new macro activateAccessibilityMenu?

Reply:
`Site Settings` is at a different place, so it cannot be activated by the macro I created.